pkg/dom0-ztools/rootfs/bin/eve

```diff
@@ -6,15 +6,15 @@ CTR_UA_CMD="ctr --namespace eve-user-apps"
 help() {
   cat <<__EOT__
 Welcome to EVE!
-  commands: enter [qube (assumed pillar)] [command (assumed sh)]
-            enter-user-app <qube>
-            exec qube command
+  commands: enter [(assumed pillar)] [command (assumed sh)]
+            enter-user-app <pillar>
+            exec pillar command
```

Comment:
As @giggsoff says below, the original code is missing angle brackets on this line.